packages/elements/src/components/ino-list-item/ino-list-item.tsx

```diff
@@ -110,7 +110,7 @@ export class ListItem implements ComponentInterface {
 
     return (
       <Host>
-        <li ref={(el) => (this.listItemEl = el)} class={listItemClasses}>
+        <li ref={(el) => (this.listItemEl = el)} class={listItemClasses} tabIndex={0}>
```

Comment:
Maybe we should expose the `tabIndex` as a Prop to the consumer, so they can decide on their one which `tabIndex` they want to use on their `listItems`.